integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/ssl/test/DynamicSSLProfilesPTTListenerTestCase.java

```diff
@@ -52,6 +52,10 @@
     private String configLocation =
             getESBResourceLocation() + File.separator + "dynamicsslprofiles" + File.separator + "pttlistener"
                     + File.separator;
+    private final String CERT_CONFIG_LOCATION =
+            FrameworkPathUtil.getSystemResourceLocation() + File.separator + "keystores" + File.separator + "products"
```

Comment:
FrameworkPathUtil.getSystemResourceLocation()  give the path with File.Separator in the end , extra separator might some time cause issue in windows.

Reply:
Done